boot/freeldr/freeldr/freeldr_i386.lds

```diff
@@ -1,6 +1,6 @@
 SECTIONS
 {
-  .text __image_base__ + __section_alignment__ :
```

Comment:
rename the file? `freeldr_i386.lds` is confusing

Reply:
x86 ?

Reply:
freeldr_gcc.lds it is